aws/manageDeployment.sh

```diff
@@ -241,19 +189,11 @@ function wait_for_deployment_execution() {
 
 function process_deployment() {
 
-  stripped_template_file="${tmp_dir}/stripped_template"
```

Comment:
This will definitely be a problem with AWS - it is why we added the stripping in the first place :-)

Need to reinstate if this script will eventually be used for AWS as well...